src/renate/updaters/avalanche/model_updater.py

```diff
@@ -61,10 +62,21 @@ def _load_learner(
             logged_metrics=self._logged_metrics,
             **self._transforms_kwargs,
         )
-        optimizer, scheduler = self._dummy_learner.configure_optimizers()
-        optimizer, scheduler = optimizer[0], scheduler[0]
-        lr_scheduler_plugin = LRSchedulerPlugin(scheduler=scheduler)
-        plugins = [lr_scheduler_plugin]
+        plugins = []
+        lr_scheduler_plugin = None
+        optimizers_scheduler = self._dummy_learner.configure_optimizers()
+        if isinstance(optimizers_scheduler, tuple):
+            optimizer, scheduler_config = optimizers_scheduler
```

Comment:
What do these two lines do?

Reply:
Return value can either be Tuple[Optimizer, Scheduler] or Optimizer. We are checking for this case and conditionally add a LRCallback.